Rabin_Karp_in_C/Rabin-Karp.c

```diff
@@ -0,0 +1,94 @@
+#include <stdio.h>
```

Comment:
Fix file name to `Rabin_Karp`.
Fix Folder name to `Rabin_Karp`

Like https://github.com/jainaman224/Algo_Ds_Notes/tree/master/Rabin_Karp